src/content/en/updates/2019/02/feb-wrap-up.md

```diff
@@ -0,0 +1,141 @@
+{# wf_featured_image: /web/updates/images/generic/monthly.png #}
+{# wf_featured_snippet: February wrap up which we look back what's been happening in Web Developer Ecosystem team.　#}
+{# wf_blink_components: N/A #}
+
+# Web Dev Ecosystem team - February wrap up {: .page-title }
+
+Welcome to the first installment of a monthly wrap up which we look back what's been happening in 
+Web Developer Ecosystem team✨
+
+We are a team of engineers and communicators who produce articles and code samples such as this 
+website [web fundamentals](/web) and our brand new portal [web.dev](https://web.dev/). You can also 
+catch our work over on [YouTube Channel](https://www.youtube.com/user/ChromeDevelopers), and 
+don't forget to follow us on [@ChormiumDev](https://twitter.com/chromiumdev) :)
+
+February is a short month but we are certainly not short on content. Let's start with big releases 
+from the team.
+
+## Releases
+### Workbox
+Hot off the press, 
+[Workbox 4.0 was released](https://github.com/GoogleChrome/workbox/releases/tag/v4.0.0) just a few 
+days ago 🎉 This release includes great new  features like `workbox-window` and  `workbox-routing`. 
+For those of you who are already using workbox, check out 
+[v3 to v4 migration guide](/web/tools/workbox/guides/migrations/migrate-from-v3). 
+Wondering how you can use them in your existing project? here is a guide to 
+[use them with bundlers of your choice](/web/tools/workbox/guides/using-bundlers). 
+Not sure what problem workbox helps to solve? check out this 
+[interview on service workers](https://www.youtube.com/watch?v=JYXXGNFJjwc) over on 
+the State of the Web show. 
+
+### lit-html and LitElement
+The team at [polymer project](https://www.polymer-project.org/) has been busy working on 
+[stable release of lit-html and LitElement](/web/updates/2019/02/lit-element-and-lit-html) - two 
+next-generation web development libraries. Do you want to try it out? start with 
+[Try LitElement](https://lit-element.polymer-project.org/try) guide 📝
+
+### Trusted Web Activities 
+With the release of Chrome 72, 
+[Trusted Web Activity (TWA)](https://blog.chromium.org/2019/02/introducing-trusted-web-activity-for.html) 
+has entered to the market! TWA let you have full screen Chrome inside of an Android Activity, which means 
+you can bring your web content into app-sphere📱 Check out this 
+[getting started guide](/web/updates/2019/02/using-twa) or read on how @svenbudak put their 
+[PWA on Google Play Store](https://medium.com/@svenbudak/this-twa-stuff-rocks-finally-i-got-my-pwa-on-google-play-store-b92fe8dae31f)
+!
+
+## What's coming next
+With Chrome 73 stable release on the horizon ([March 12](https://www.chromestatus.com/features/schedule)
+), we have lots of exciting features to cover!
+ 
+**V8** - Chrome's JavaScript engine has a bunch of updates including `Object.fromEntries` and 
+`String.prototype.matchAll`. Be sure to check the [release notes here](https://v8.dev/blog/v8-release-73)
```

Comment:
Nit/Best Practice: instead of using "release notes here", try "Check out the [v8 release notes]" - that makes it more clear about what the link links to, and helps search engines index stuff.